tests/requirements/base.txt

```diff
@@ -1,3 +1,4 @@
+argon2-cffi >= 16.0.0
```

Comment:
these dependencies are also mentioned in docs/internals/contributing/writing-code/unit-tests.txt